### ISSUE TITLE
PostBIS support?

### ISSUE THREAD

**Issue by auvipy** [biopython/biopython#719]

**Comment by peterjc** [comment]
What is PostBIS and how would it fit into Biopython?

**Comment by auvipy** [comment]
postbis is an bio extension to postgresql

**Comment by peterjc** [comment]
You mean https://colab.mpi-bremen.de/wiki/display/pbis/PostBIS

Do you think this would be useful in the ``BioSQL`` part of Biopython?

**Comment by auvipy** [comment]
yes

**Comment by auvipy** [comment]
those data types could be implemented using psycopg2

**Comment by peterjc** [comment]
This would mean schema changes to BioSQL specific to PostgreSQL, I suggest you raise this on https://github.com/biosql/biosql

**Comment by auvipy** [comment]
thats a perl tool!!

**Comment by peterjc** [comment]
No, BioSQL is an SQL database schema (a set of tables designed for storing
annotated sequence data from sources like NCBI GenBank) for several
different databases including PostgreSQL, MySQL, SQLite.

BioSQL does come with some scripts which happen to be written in Perl for
tasks like importing the NCBI taxonomy tree into the species tables.

There are separate BioSQL interfaces in BioPerl, Biopython, BioRuby and
BioJava which are intended to all be interoperable (they should be able to
read and write to a shared BioSQL database).
